airbyte-integrations/connectors/destination-snowflake/src/main/kotlin/io/airbyte/integrations/destination/snowflake/sql/SnowflakeColumnUtils.kt

```diff
@@ -60,10 +60,10 @@ class SnowflakeColumnUtils {
         when (type) {
             BooleanType -> SnowflakeDataType.BOOLEAN.typeName
             DateType -> SnowflakeDataType.DATE.typeName
-            IntegerType -> SnowflakeDataType.INTEGER.typeName
-            NumberType -> SnowflakeDataType.NUMBER.typeName
+            IntegerType -> SnowflakeDataType.NUMBER.typeName
```

Comment:
Is it what we do in the legacy connector? It looks wrong. Should we be explicit about the precision (NUMBER(38, 0))